sdk/ml/azure-ai-ml/azure/ai/ml/entities/_compute/compute_instance.py

```diff
@@ -230,10 +235,14 @@ def _to_rest_object(self) -> ComputeResource:
             subnet=subnet_resource,
             ssh_settings=ssh_settings,
             personal_compute_instance_settings=personal_compute_instance_settings,
-            idle_time_before_shutdown=self.idle_time_before_shutdown,
+            idle_time_before_shutdown=f"PT{self.idle_time_before_shutdown_minutes}M",
```

Comment:
Nit: this works for numbers bigger than 60 right?

Reply:
Yep -- up to PT4320M, which is 72 hours.